### ISSUE TITLE
Essential plots?

### ISSUE THREAD

**Issue by trentdupuy** [pkgw/mcmc-reporting#1]
Thinking about what plots should always be shown... I propose that histograms should always be shown for all parameters (i.e., both nuisance ones and interesting ones) but correlation plots need only be shown of the interesting ones.  For example, in visual binary orbit fitting there are 7 parameters, and I'd say only 3 of these are interesting (a, P, e) while 4 are nuisance parameters with obvious priors (i, Omega, omega, and lambda_ref or T_0).  This approach would save us from behemoth triangle plots that can be very hard to parse visually.

**Comment by pkgw** [comment]
Hmmm. Should we even recommend that histograms always be shown? If a nuisance parameter has a boring posterior distribution (i.e. very Gaussian-looking), I'm comfortable with that just being reported verbally. I guess part of this is that I'm worrying about these recommendations "scaling" to analyses with many many parameters, but I suppose that's not the case I should worry about, since folks doing that kind of work are probably fairly "advanced" users.

By the way, you're talking about histograms for parameters marginalizing over all others, right? In the text I'm trying to make a point of explicitly labeling various distributions as being marginalized or not.

**Comment by trentdupuy** [comment]
True, I was envisioning a manageable number of parameters, and yes marginalized posteriors.  I don't know the argument for not marginalizing when reporting single parameter results.  I suppose I could adjust my statement to say that histograms should be shown when there are relatively few parameters (<~10), otherwise intervals demonstrating that posteriors are roughly symmetric and Gaussian would be sufficient.  But anything not roughly Gaussian should perhaps be shown as a histogram?

This reminds me also that I feel like both 1 and 2 sigma intervals should always be reported.  Among other reasons, that is at least one way to see if something is roughly Gaussian.
